frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -388,6 +389,7 @@ function ReaderHighlight:addToMainMenu(menu_items)
             checked_func = function()
                 return G_reader_settings:readSetting("default_highlight_action", "ask") == v[2]
             end,
+            radio = true,
```

Comment:
And separator = true for the opacity thing? (If this is the right place; just taking a look really quickly.)

![Screenshot_2022-02-01_13-12-55](https://user-images.githubusercontent.com/202757/151966411-7fd07355-4374-4d0a-be92-eb8e36998310.png)

Reply:
Opacity relates to and is enabled with `Lighten` only, I'm afraid the separator may break the relation.

Reply:
Given that it's not directly adjacent to Lighten, the visual relation is already not obvious.
And there's the fact that it's enabled only when Lighten is selected, so the relation is made obvious by its dim/black state.
And/or it could be named "Ligten/lightening opacity" :) ?
Anyway, no strong opinion whether a separator is best or not.

Reply:
The separator relates to the fact that it's not really clear where the radio stuff ends or not. That makes is slightly different from the logic behind a "regular" separator.